src/Microsoft.Health.Dicom.Core/Features/Workitem/WorkitemOrchestrator.cs

```diff
@@ -190,12 +190,21 @@ public async Task<DicomDataset> GetWorkitemBlobAsync(WorkitemInstanceIdentifier
             .ConfigureAwait(false);
     }
 
+    /// <inheritdoc />
+    public async Task<DicomDataset> RetrieveWorkitemAsync(WorkitemInstanceIdentifier workitemInstanceIdentifier, CancellationToken cancellationToken = default)
+    {
+        EnsureArg.IsNotNull(workitemInstanceIdentifier, nameof(workitemInstanceIdentifier));
+
+        return await TryGetWorkitemBlobAsync(workitemInstanceIdentifier, cancellationToken)
```

Comment:
Should we use try get or just get? Since you are already handling workitemnotfoundexception in the catch. We can remove the null check in the try